dbt_common/contracts/config/base.py

```diff
@@ -115,6 +115,28 @@ def same_contents(cls, unrendered: Dict[str, Any], other: Dict[str, Any]) -> boo
         "object": ["snapshot_meta_column_names"],
     }
 
+    @classmethod
+    def update_from(
+        cls, dct: Dict[str, Any], data: Dict[str, Any], adapter_config_cls: Type[BaseConfig]
```

Comment:
naming nit: is there naming here that would better signify the difference between `dct` and `data`?

Reply:
Do you have any suggestions?   orig_dict, new_dict?